sdk/python/kfp/client/client.py

```diff
@@ -1132,6 +1137,15 @@ def create_run_from_pipeline_package(
             enable_caching=enable_caching,
             service_account=service_account,
         )
+    
+        link = f"{self._get_url_prefix()}/#/runs/details/{run_info.id}"
+        if self._is_ipython():
+            import IPython
+            html = (f'<a href="{link}" target="_blank" >Run details</a>.')
+            IPython.display.display(IPython.display.HTML(html))
+        else:
+            print(f"Run details: {link}")
+
```

Comment:
Didn't notice this before, but this method calls `.run_pipeline`, which also has link printing. We can omit link printing here.

Reply:
Removed it! 👍